mcs/class/referencesource/mscorlib/system/activator.cs

```diff
@@ -184,9 +184,14 @@ static public Object CreateInstance(Type type)
                                   null,
                                   ref stackMark);
         }
-            
+
+        public static object CreateInstance(Type type, bool nonPublic)
+        {
+            return CreateInstance(type, nonPublic, wrapExceptions: true);
+        }
+
         [MethodImplAttribute(MethodImplOptions.NoInlining)] // Methods containing StackCrawlMark local var has to be marked non-inlineable
-        static public Object CreateInstance(Type type, bool nonPublic)
+        internal static object CreateInstance(Type type, bool nonPublic, bool wrapExceptions)
```

Comment:
Should be just private

Reply:
Before this change RuntimeType called `Activator.CreateInstance(Type type, bool nonPublic)` when a constructor has no arguments. After this change I needed a way to propagate the DoNotWrapExceptions flag, so I added this internal overload.

Reply:
As it's used by CreateInstance only, it does have to be `internal`